src/open_inwoner/cms/cases/tests/test_htmx.py

```diff
@@ -423,7 +423,7 @@ def test_cases(self, m):
         page.goto(self.live_reverse("cases:index"))
 
         # case title
-        case_title = page.get_by_role("link", name=self.zaaktype["omschrijving"])
+        case_title = page.get_by_text("Coffee zaaktype")
```

Comment:
This is not right, given that `self.zaaktype["omschrijving"]` = `"Coffee zaaktype"`.

The problem is `aria-label="{% trans "Bekijk aanvraag" %}"` in the link of the case. The accessible name for the link is given by the content of the `h4` element, hence Playwright gets confused and thinks the `name` for the `link` role is "Bekijk aanvrag", the content of `aria-label`. If you replace `aria-label` with `aria-labelledby` and link it to the `h4`, the test works. Two more comments:
- Strictly speaking, the content of the link is much more than "Coffee zaaktype"; it contains the information of all the list elements inside `<a>....</a>`. I don't know if it's still appropriate to use `aria-labelledby` in this case, linking it to only part of the content, though my intuition is that it's OK.
- If you change the html as suggested, you should also change it for the open submission, which has the same structure. We don't have a separate test for those, but the situation is the same and should be handled accordingly.

Reply:
Ah yes, I actually need to rethink this. An aria-label is not needed in case of an anchor-surrounding/overriding-everything but we probably still want a separate label for the button inside the anchor.

Reply:
@pi-sigma After deeper consideration: I've removed the title attributes since those are actually very bad for accessibility - also the aria-labels are not necessary here because the screenreaders have access to all texts and can read everything (except for the purposefully hidden icons).
(I should start removing unnecessary labels throughout the entire project).
If the component follows correct HTML you don't need aria-labels, unless you wish to override something.